tutorial/dashbio_component_examples.py

```diff
@@ -38,10 +40,17 @@
             'param_name': 'Color Scales',
             'description': 'Change the colors used for the heatmap.',
             'code': '''from six.moves.urllib import request as urlreq
+from six import PY3
 import dash_bio as dashbio
 
 
-data = urlreq.urlopen("https://raw.githubusercontent.com/plotly/dash-bio/master/tests/dashbio_demos/sample_data/alignment_viewer_p53.fasta").read().decode('utf-8')
+data = urlreq.urlopen(
+    "https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/" +
```

Comment:
```suggestion
    'https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/' +
```
because we have single quotes from line 117 onwards